webapp/src/components/mobilitymap/InstructionsModal.js

```diff
@@ -0,0 +1,246 @@
+                <h5>Android 9</h5>
+                <ol>
+                  <li>Open your device&#39;s Settings app</li>
+                  <li>
+                    Tap <b>Security &amp; Location</b> &gt; <b>Location.</b>
+                    <ul>
+                      <li>
+                        If you have a work profile, tap <b>Advanced.</b>
+                      </li>
+                    </ul>
+                  </li>
+                  <li>
+                    <b>Turn Location on</b>: Tap <b>Location.</b>
+                  </li>
+                </ol>
+                <h5>Android 4.4 to 8.1</h5>
+                <ol>
+                  <li>Open your phone&#39;s Settings app.</li>
+                  <li>
+                    Tap <b>Security &amp; Location</b>. If you don&#39;t see &#34;Security
+                    &amp; Location,&#34; tap <b>Location</b>.
+                  </li>
+                  <li>
+                    Tap <b>Mode</b>. Then pick:
+                  </li>
+                </ol>
+                <ul>
+                  <li>
+                    <b>High accuracy:</b> Use GPS, Wi-Fi, mobile networks, and sensors to
+                    get the most accurate location. Use Google Location Services to help
+                    estimate your phone&#39;s location faster and more accurately.
+                  </li>
+                  <li>
+                    <b>Battery saving:</b> Use sources that use less battery, like Wi-Fi
+                    and mobile networks. Use Google Location Services to help estimate
+                    your phone&#39;s location faster and more accurately.
+                  </li>
+                  <li>
+                    <b>Device only:</b> Use only GPS. Don&#39;t use Google Location
+                    Services to provide location information. This can estimate your
+                    phone&#39;s location more slowly and use more battery.
+                  </li>
+                </ul>
+                <h5>Android 4.1 to 4.3</h5>
+                <ol>
+                  <li>Open your phone&#39;s Settings app.</li>
+                  <li>
+                    Under &#34;Personal&#34;, tap <b>Location access</b>.
+                  </li>
+                  <li>
+                    At the top of the screen, turn <b>Access to my location</b> on
+                  </li>
+                </ol>
+              </Accordion.Body>
+            </Accordion.Item>
+            <Accordion.Item eventKey="ios">
+              <Accordion.Header>IOS Device</Accordion.Header>
+              <Accordion.Body>
+                <ol>
+                  <li>
+                    Go to <b>Settings &#62; Privacy &#62; Location Services</b>
+                  </li>
+                  <li>
+                    Make sure that <b>Location Services</b> is turned on
+                  </li>
+                </ol>
+              </Accordion.Body>
+            </Accordion.Item>
+          </Accordion>
```

Comment:
Is there a message for unable to identify? (Opera, etc)

Reply:
None atm. If no ua detection, we return and fallback to user following instructions in modal. But it's possible to add a no-detection message just so they're aware. What's your opinion on this?

Reply:
Oh, so everything is collapsed, and they choose the one they need? That seems good.

Reply:
Actually none would not be collapsed if no browser/device detection. But yes same concept.